### ISSUE TITLE
fftw: enable for aarch64

### ISSUE THREAD

**Issue by kmilos** [msys2/MINGW-packages#11207]
Seems there is quite a bit of delicate configure option cleanup to be done (I don't have a handle on it all yet..)

**Comment by MehdiChinoune** [comment]
These are the ARM instructions:
```
  --enable-neon           enable ARM NEON optimizations
  --enable-armv8-pmccntr-el0
                          enable the cycle counter on ARMv8 via the
                          PMCCNTR_EL0 register (see README-perfcounters for
                          details and mandatory instructions)
  --enable-armv8-cntvct-el0
                          enable the cycle counter on ARMv8 via the CNTVCT_EL0
                          register (see README-perfcounters for details and
                          mandatory instructions)
  --enable-armv7a-cntvct  enable the cycle counter on Armv7a via the CNTVCT
                          register (see README-perfcounters for details and
                          mandatory instructions)
  --enable-armv7a-pmccntr enable the cycle counter on Armv7a via the PMCCNTR
                          register (see README-perfcounters for details and
                          mandatory instructions)
```
Enabling all of them won't hurt as FFTW uses dynamic loading.

**Comment by MehdiChinoune** [comment]
Fixed by #11981
